scripts/biomedical/VirtualMetabolicHuman/format_reaction_subsystem.py

```diff
@@ -0,0 +1,30 @@
+import sys
+import os
+import pandas as pd
+import numpy as np
+import csv
+
+def main():
+    file_input = sys.argv[1]
+    file_output = sys.argv[2]
+
+    df = pd.read_csv(file_input)
+    #Get unique subsystem values
```

Comment:
Prefer keeping the comments close to where the logic is. This could move to the line before the unique() call?

Reply:
Fixed it, thank you !